notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>Typo: ponits -&gt; points</p><p> </p><p>NIT: to be slightly more precise: </p><p>if you move a tangent vector from one point to another (i.e. from one tangent space to another), the vector will not...</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>agree</p>